api/src/main/java/com/netflix/iceberg/expressions/InclusiveMetricsEvaluator.java

```diff
@@ -53,12 +54,20 @@ private MetricsEvalVisitor visitor() {
     return visitors.get();
   }
 
-  public InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
+  InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
     this.schema = schema;
     this.struct = schema.asStruct();
+    this.caseSensitive = true;
```

Comment:
Same as the previous comment about constructor overloading we could probably have this constructor rewritten as `this(schema, unbound, true)`?

Reply:
Agreed.